.gitignore

```diff
@@ -4,6 +4,7 @@ bin/wasm2asm
 bin/s2wasm
 bin/wasm-as
 bin/wasm-dis
+lib/libbinaryen-c.so
```

Comment:
The bin directory is all specified individually so I was did the same thing. I'll defer to those who know the code better than I do.